tests/test_chmod.rs

```diff
@@ -59,15 +59,15 @@ fn run_tests(tests: Vec<TestCase>) {
 #[test]
 fn test_chmod_octal() {
     let tests = vec!{
-        TestCase{args: vec!{"0700",  TEST_FILE}, before: 0o000, after: 0o700},
-        TestCase{args: vec!{"0070",  TEST_FILE}, before: 0o000, after: 0o070},
-        TestCase{args: vec!{"0007",  TEST_FILE}, before: 0o000, after: 0o007},
-        TestCase{args: vec!{"-0700", TEST_FILE}, before: 0o700, after: 0o000},
-        TestCase{args: vec!{"-0070", TEST_FILE}, before: 0o060, after: 0o000},
-        TestCase{args: vec!{"-0007", TEST_FILE}, before: 0o001, after: 0o000},
-        TestCase{args: vec!{"+0100", TEST_FILE}, before: 0o600, after: 0o700},
-        TestCase{args: vec!{"+0020", TEST_FILE}, before: 0o050, after: 0o070},
-        TestCase{args: vec!{"+0004", TEST_FILE}, before: 0o003, after: 0o007},
+        TestCase{args: vec!{"0700",  TEST_FILE}, before: 0o100000, after: 0o100700},
+        TestCase{args: vec!{"0070",  TEST_FILE}, before: 0o100000, after: 0o100070},
+        TestCase{args: vec!{"0007",  TEST_FILE}, before: 0o100000, after: 0o100007},
+        TestCase{args: vec!{"-0700", TEST_FILE}, before: 0o100700, after: 0o100000},
+        TestCase{args: vec!{"-0070", TEST_FILE}, before: 0o100060, after: 0o100000},
+        TestCase{args: vec!{"-0007", TEST_FILE}, before: 0o100001, after: 0o100000},
+        TestCase{args: vec!{"+0100", TEST_FILE}, before: 0o100600, after: 0o100700},
+        TestCase{args: vec!{"+0020", TEST_FILE}, before: 0o100050, after: 0o100070},
+        TestCase{args: vec!{"+0004", TEST_FILE}, before: 0o100003, after: 0o100007},
```

Comment:
Given that this breaks the test suite for stable, I think it would be a good idea to use the `rustc_version` crate to change the expected before and after values depending on which version of the compiler is being used.

Reply:
Not sure what you mean - for 1.22 (current stable) the `chmod` tests work fine. The tests for `install` still fail, though. I guess it would make sense to fix it as part of this change?

Reply:
Ah, sorry, I guess I meant 1.21 (I did not notice the stable version had changed).  However, given that we are now testing 1.20 as the minimum required version of Rust, can you either add the version checks or bump the minimum required version of the compiler?

Thanks for fixing `install` as well. :)